process_request/helpers.py

```diff
@@ -3,6 +3,164 @@
 from rapidfuzz import fuzz
 
 
+def get_container_indicators(instance):
+    """Takes ArchivesSpace JSON instance data with resolved top containers and returns a container indicator.
+
+    Args:
+        instance (dict): ArchivesSpace instance information that has resolved top containers and digital objects.
+
+    Returns:
+        string: A concatenated string containing the container type and container indicator, or digital object title.
+    """
+    if instance.get("instance_type") == "digital_object":
+        return "Digital Object: " + instance.get("digital_object").get("_resolved").get("title")
+    else:
+        return instance.get("sub_container").get("top_container").get("_resolved").get("type").capitalize() + ' ' + instance.get("sub_container").get("top_container").get("_resolved").get("indicator")
+
+
+def get_file_versions(digital_object):
+    """Takes ArchivesSpace digital object json and returns a file version.
+
+    Args:
+        digital_object (dict): Resolved json of an ArchivesSpace digital object.
+
+    Returns:
+        string: all file version uris associated with the digital object, separated by a comma.
+    """
+    versions = []
+    [versions.append(v.get("file_uri")) for v in digital_object.get("file_versions")]
```

Comment:
I think you were trying to do this:

```suggestion
    versions = [v.get("file_uri")) for v in digital_object.get("file_versions")]
```

Reply:
I forgot that you could assign to a list with list comprehension. So yeah, this is much simpler, thanks!